bubblesub/data/menu.conf

```diff
@@ -192,6 +180,26 @@ A&udio
     -
     Shift selected subtitles...|sub-shift --no-align
 
+&View
```

Comment:
Why did you move View to the right

Reply:
I thought it would be useful to move the commands that are related to the presentation of the widgets to the right in order to separate them from the other ones. In addition, I'm going to add the preferences dialog box there, so I think it's the best place on the right.